app/models/configuration_script.rb

```diff
@@ -1,3 +1,17 @@
 class ConfigurationScript < ActiveRecord::Base
-  belongs_to :configuration_manager, :class_name => 'ManageIQ::Providers::ConfigurationManager'
+  belongs_to  :manager,
+              :class_name  => 'ManageIQ::Providers::ConfigurationManager',
+              :foreign_key => :configuration_manager_id
+  include ProviderObjectMixin
+
+  def run(vars=nil)
+    extra_vars = vars.reverse_merge(variables)
```

Comment:
@syncrou 
I am calling run as 
sc.run({'pkg' => 'httpd', 'sleep' => 3}.to_json)
When the job runs in Ansible Tower the job looks like
 "extra_vars": "{\"host\": \"ManageIQ(TinaF)\", \"sleep\": 180, \"pkg\": \"abrt\", \"user\": \"root\"}",
The sleep and the pkg attributes didn't get updated